.rubocop.yml

```diff
@@ -8,6 +8,7 @@ AllCops:
     - '**/vendor/**/*'
     - 'actionpack/lib/action_dispatch/journey/parser.rb'
     - 'railties/test/fixtures/tmp/**/*'
+    - 'actionmailbox/test/dummy/**/*'
```

Comment:
Since we haven't excluded all cops for `activestorage/test/dummy/**/**`, probably we should consider turning on all the cops for `actionmailbox/test/dummy/**/*` to keep consistency? We can just run `rubocop -a` to fix all offenses under the `dummy` directory.